### PR TITLE
Add the PDFStreamDumper package

### PR DESCRIPTION
**Pull request by d35ha**
Closes https://github.com/mandiant/VM-Packages/issues/213

### CODE REVIEW

File: packages/pdfstreamdumper.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    $category = 'PDF'
+
+    $exeUrl = 'http://sandsprite.com/flare_vm/PDFStreamDumper_Setup_C26068186F63DCCE9CC57502BE742C728110EAB07570C319A0D7D10587A6E22D.exe'
+    $exeSha256 = 'c26068186f63dcce9cc57502be742c728110eab07570c319a0d7d10587a6e22d'
+
+    $toolDir = Join-Path ${Env:RAW_TOOLS_DIR} $toolName
+    $executablePath = (Join-Path $toolDir "$toolName.exe")
+    VM-Install-With-Installer -toolName $toolName -category $category -fileType "EXE" -silentArgs "/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP- /Dir=`"$($toolDir)`"" -executablePath $executablePath -url $exeUrl -sha256 $exeSha256
+
+    $desktopShortcut = Join-Path "C:\Users\$Env:UserName\Desktop" "$toolName.exe.lnk"
```
Comment (Ana06) [inline packages/pdfstreamdumper.vm/tools/chocolateyinstall.ps1:15]:
```suggestion
    $desktopShortcut = Join-Path ${Env:UserProfile} "Desktop\$toolName.exe.lnk"
```

**Review by Ana06 — COMMENTED** [review]
Just a minor comment and we are good to go, thanks @d35ha! 🚀

File: packages/pdfstreamdumper.vm/tools/chocolateyinstall.ps1 (hunk 1 above)
Comment (d35ha) [reply packages/pdfstreamdumper.vm/tools/chocolateyinstall.ps1]: Done.

**Review by Ana06 — APPROVED** [review]

**Comment by Ana06** [comment]
Great work @d35ha!

**Pull request merged**